proxy-server/minikube-test.yaml

```diff
@@ -38,7 +32,7 @@ spec:
       - name: PROXY_LISTEN_PORT
         value: "8080"
       - name: PROXY_TARGET_URL
-        value: "http://localhost:80"
+        value: "http://web-resource:80"
       - name: PROXY_FALLBACK_URL
         value: "http://localhost:8888/"
       - name: PROXY_DEBUG
```

Comment:
@forana Alex, you tested it all inside the cluster, right?  It would not be working if you hit proxy from outside, because proxy_fallback is where browser is redirected and it's surely not resolving if you are outside.

Reply:
@andrewelkin I tested using https://github.com/saturncloud/images/blob/master/proxy-server/minikube-test.sh - there's a port-forward in there that lets it work.